lib/googleanalytics.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
+var config = require('./config');
+
+var GoogleAnalyticsBasicTag = React.createClass({
+  render: function() {
+    if (config.GA_ACCOUNT) {
+      return (
+        <script dangerouslySetInnerHTML={{
+          __html: "(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ \
```

Comment:
Oh, so by putting the GA code into an actual module, I meant actually taking the value of `__html` and turning it into its own module. For instance, something like this:

```javascript
exports.initialize = function() {
  if (!config.GA_ACCOUNT) return;

  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', config.GA_ACCOUNT, 'auto');
  ga('send', 'pageview');
};
```

Then the first line of `routes.jsx`'s `run()` function could just be `ga.initialize()`, and we wouldn't have to deal with putting a script tag with "dangerous inner HTML" in at all... But we could either land this and I could issue a PR, or we could just go with your version.